examples/bert/bert_ptq_cpu.json

```diff
@@ -49,9 +49,12 @@
         },
         "transformers_optimization": {
             "type": "OrtTransformersOptimization",
+            "disable_search": true,
             "config": {
-                "float16": false,
```

Comment:
@trajepl , is this ok to remove only_onnxruntime?

Reply:
It is fine, the default value is False. Just ensure there is at least one model conducted from Olive run.